recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ source:
 
 build:
   number: {{ build }}
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  script: pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
```

Comment:
I would add `pip` to `host`, and `pip check` to `tests`. It's up to you

Reply:
The home ULR isn't working, so update to https://github.com/linsomniac/python-memcached

Reply:
license PSF-2.0

Reply:
addressed, thank you!